packages/graphiql-react/src/explorer/components/search.tsx

```diff
@@ -60,20 +50,16 @@ export function Search() {
 
   const navItem = explorerNavStack.at(-1)!;
 
-  const onSelect = useCallback(
-    (def: TypeMatch | FieldMatch) => {
-      push(
-        'field' in def
-          ? { name: def.field.name, def: def.field }
-          : { name: def.type.name, def: def.type },
-      );
-    },
-    [push],
-  );
-  const isFocused = useRef(false);
-  const handleFocus: FocusEventHandler = useCallback(e => {
-    isFocused.current = e.type === 'focus';
-  }, []);
```

Comment:
Hi 👋

Was there a reason to remove this? This might be related to the following regression: https://github.com/graphql/graphiql/issues/3842

Reply:
Hi, it still present https://github.com/graphql/graphiql/blob/1d598a9439a0cb265a72b830e68aac71e79337f1/packages/graphiql-react/src/explorer/components/search.tsx#L60-L62

I will take a look at your issue soon!